keymaps/linux.cson

```diff
@@ -51,8 +51,8 @@
   'ctrl-shift-tab': 'pane:show-previous-item'
   'ctrl-pageup': 'pane:show-previous-item'
   'ctrl-pagedown': 'pane:show-next-item'
-  'ctrl-shift-up': 'core:move-up'
```

Comment:
Would you mind re-mapping https://github.com/dantaex/atom/blob/patch-2/keymaps/linux.cson#L115-L116 to `core:move-up`?